rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -155,6 +223,8 @@ def __init__(self):
             if len(element.strip().strip("'")) > 0
         ]
 
+<<<<<<< HEAD
+=======
```

Comment:
Source control conflict markers and a bunch of other commits showing up in this pull request. I think you should try to fix the rebase.

Reply:
Oh, my mistake. Now I think I resolved those conflict.